packages/api/src/external/ehr/athenahealth/command/get-patient.ts

```diff
@@ -204,7 +230,9 @@ async function getPatientByDemo({
     const patient = await singleGetMetriportPatientByDemo({ cxId, demo });
     if (patient) patients.push(patient);
   } catch (error) {
-    const msg = `Failed to get patient by demo. Cause: ${errorToString(error)}`;
+    const msg = `Failed to get patient by demo. cxId ${cxId} athenaPracticeId ${athenaPracticeId} athenaPatientId ${athenaPatientId}. Cause: ${errorToString(
```

Comment:
you already set the cxId, athenaPracticeId, athenaPatientId on the log definition at line 61, and now youre passing it to this func, so this is redundant

Reply:
This is for sentry. The log message would appear in AWS logs, but sentry. Normally you'd pass an "extras" object with these values (and exclude them from the msg) but I'm aggregating all the errors into a single sentry alert, so I need to make sure they're in the message itself.

Reply:
This one I just made more efficient as the incoming log already has the details info.